0018-connector-risk-mitigations/0018-connector-risk-mitigations.md

```diff
@@ -17,25 +17,28 @@ The main risk connectors face in Interledger is being unable to fulfill the inco
 Possible mitigations include:
 
 * **Connecting Reliable Ledgers** - Connectors should choose the ledgers they connect in part based on the reliability and speed of fulfillment notifications and the time the ledgers require to process fulfillments once they are submitted
-* **Adjusting Transfer Expiry Window** - Connectors determine the window of time they require between when the incoming and outgoing transfers expire. Connectors should set this window such that they are confident that they will be able to deliver the fulfillment in time even if the outgoing transfer is executed at the last possible moment.
```

Comment:
This is still relevant. 

How two connectors reconcile and/or enforce the settlement of their Interledger accounts is outside the scope of the protocol but the protocol does still expect them to respect the expiry times and deliver fulfillments before the corresponding prepare expires.

Reply:
Maybe clarify that Transfer Expiry Window is no longer dependent on execution of on-ledger transaction, just on the amount of time it takes to receive a fulfill from their bilateral party.